src/components/Toggle/Toggle.scss

```diff
@@ -0,0 +1,87 @@
+$toggle-track-length: rem(46px);
+$toggle-track-height: rem(24px);
+$toggle-thumb-size: rem(20px);
+$disabled-opacity: 0.6;
```

Comment:
This should be consistent with your naming convention, so `toggle-disabled-opacity`.